src/Lecture1.hs

```diff
@@ -119,4 +131,8 @@ and lower than 6 elements (4, 5, 6, 7, 8 and 9).
 
 🕯 HINT: Use recursion to implement this function.
 -}
-lowerAndGreater n list = error "TODO"
+lowerAndGreater :: Int -> [Int] -> [Char]
+lowerAndGreater n list = concat [show n, " is greater than ", show (calc (< n) list), " elements and lower than ", show (calc (> n) list), " elements"] where
```

Comment:
That's correct! 💯 

However, with this solution, you will walk through the list twice.
You can write it more performant with only one traversal of the list 🙂
It's desirable to implement functions that traverse lists single time and consume each element only once. In that case, the composition of such functions will have nice streaming and lazy properties 🦥

Reply:
You are quite right. I had in my mind this point initially but forget after the first naive implementation. Would be better to use foldl but I wanted to implement loop on my own. In any case I rewrote it.